docs/src/components/Combination.js

```diff
@@ -117,7 +117,7 @@ export default function Combination({
             alignItems="center"
           >
             {showValues && (
-              <Box marginBottom={2}>
+              <Box marginBottom={2} id={`${id || ''}-label-${i}`}>
```

Comment:
@ayeshakmaz this is a bit of magic - using the `-label-`, would there be a way to avoid that? E.g. not using `accessibilityLabelledBy`

Reply:
Switched this up to use a new prop on Combinations `useLabelForValues`, let me know what you think